authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/thirdparty/keycloak/KeycloakService.kt

```diff
@@ -77,6 +74,36 @@ class KeycloakService {
         return keycloak.realm(keycloakRealm).users().search(email).first().id
     }
 
+    fun createKeycloakUserAndOrganization(
+        organizationId: String,
+        userEmail: String,
+        userFirstName: String,
+        userLastName: String,
+        userOrganizationRole: OrganizationRole,
+        userPassword: String?
+    ): String {
+        // To avoid syncing issues, we assume our DB to be the source of truth, so we need to delete the potentially existing user in KC
+        val maybeExists = getUserIdByEmail(userEmail)
```

Comment:
lol I would rename it to 'userId' since it's a nullable String which can contain value or not

Reply:
But I declare userId 2 lines further down... So if I call this throwaway variable `userId`, I would need to either make it a `var` (🤮) or use another name

Reply:
My suggestions are 'userIdIfExists' or 'foundUserId'